service/app/util/Message.java

```diff
@@ -0,0 +1,19 @@
+package util;
+
+/**
+ * This class will contains all the messages used inside telemetry micro service.
+ *
+ * @author Manzarul
+ */
+public class Message {
```

Comment:
The names of variables should clearly tell whether the message is an Error, Warning, Success, Info, etc.

Reply:
Variable names updated.